dist/boards/slstk3400a/include/periph_conf.h

```diff
@@ -147,11 +147,21 @@ static const uart_conf_t uart_config[] = {
         .loc = USART_ROUTE_LOCATION_LOC4,
         .cmu = cmuClock_USART1,
         .irq = USART1_RX_IRQn
+    },
+    {
+        .dev = LEUART0,
+        .rx_pin = GPIO_PIN(PD, 5),
+        .tx_pin = GPIO_PIN(PD, 4),
+        .loc = LEUART_ROUTE_LOCATION_LOC0,
+        .cmu = cmuClock_LEUART0,
+        .irq = LEUART0_IRQn
     }
 };
 
 #define UART_NUMOF          ARRAY_SIZE(uart_config)
-#define UART_0_ISR_RX       isr_usart1_rx
+#define UART_0_ISR_RX       isr_usart0_rx
```

Comment:
I think this one is wrong: `UART0` maps to `isr_usart_1_rx` and `UART1` to `isr_leuart0`